src/components/Table/GridTable.test.tsx

```diff
@@ -44,7 +44,13 @@ type ChildRow = { kind: "child"; id: string; name: string };
 type GrandChildRow = { kind: "grandChild"; id: string; name: string };
 type NestedRow = HeaderRow | ParentRow | ChildRow | GrandChildRow;
 
+// I tried https://github.com/keiya01/react-performance-testing#count-renders but
+// it didn't work with our fake timers, so this is easier for now.
+let renderedNameColumn: string[] = [];
+beforeEach(() => (renderedNameColumn = []));
```

Comment:
Adding some super-basic re-render tracking, so we can assert against re-rendering behavior.

Granted, this assumes we can instrument our components with code that manually writes to a global variable, which we probably shouldn't do in production code per se, but for this it works just fine.